packages/@o3r/test-helpers/package.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "@o3r/test-helpers",
+  "private": true,
+  "version": "0.0.0-placeholder",
+  "peerDependencies": {
+    "@angular-devkit/core": "^16.0.0",
+    "@angular-devkit/schematics": "^16.0.0",
+    "@angular/common": "^16.0.0",
```

Comment:
is the `@angular/*` needed by this package or by `ng-add` schematics only?
If it is only for schematics, it should be optional as well